packages/rx_bloc/README.md

```diff
@@ -228,6 +229,7 @@ class HomePage extends StatelessWidget {
 [rx_bloc_provider_lnk]: https://pub.dev/documentation/flutter_rx_bloc/latest/flutter_rx_bloc/RxBlocProvider-class.html
 [rx_bloc_builder_lnk]: https://pub.dev/documentation/flutter_rx_bloc/latest/flutter_rx_bloc/RxBlocBuilder-class.html
 [booking_app_sample_lnk]: https://github.com/Prime-Holding/rx_bloc/tree/master/examples/booking_app
+[todo_sample_lnk]: https://github.com/Prime-Holding/rx_bloc/tree/develop/examples/todoapp
```

Comment:
All the other projects are linked to the master branch except for this one. Should this one do the same?

Reply:
It's not in the master yet. @StanevPrime should I the link to the master to a non existing project?

Reply:
The todo app has been merged to the master branch

Reply:
Done